### PR TITLE
Support working without ZetaSQL

### PR DESCRIPTION
**Pull request by pgrivachev**
- Supported working without ZetaSQL
- Added pipeline for Windows (tests are temporary disabled)
- Now we try to parse manifest.json earlier to be able to have some dbt features working in case this file already exists
- Now we do `dbt compile` on file open event if manifest.json doesn't exist at this point
- Fixed problem with feature detection (for example sometimes we had `dbtRpcGlobalVersion === undefined` even if `dbt-rpc --version` command exited with code 0

### CODE REVIEW

File: server/src/test/ZetaSqlWrapper.spec.ts [hunk 1]
```diff
@@ -24,44 +26,46 @@ describe('ZetaSqlCatalogTest', () => {
...
+      assertThat(datasets.get(expectedDataSet)?.name, expectedDataSet);
     }
 
-    const tables = expectedDataSet ? datasets?.get(DATA_SET)?.tables : (zetaSqlCatalog as any).catalog.tables;
-    assertThat(tables?.size, 1);
-    assertThat(tables?.get(table)?.name, table);
+    const tables = expectedDataSet ? datasets?.get(DATA_SET)?.tables : simpleCatalog.tables;
+    assert.ok(tables);
+    assertThat(tables.size, 1);
+    assertThat(tables.get(table)?.name, table);
```
Comment (fivetran-dmitryboykov) [inline server/src/test/ZetaSqlWrapper.spec.ts:63]: Try:
```
assertThat(tables, defined());
assertThat(tables, hasSize(1));
assertThat(tables?.get(table)?.name, table);
```

Comment (fivetran-dmitryboykov) [reply server/src/test/ZetaSqlWrapper.spec.ts:63]: Review others in this file.

**Review by fivetran-dmitryboykov — CHANGES_REQUESTED** [review]
Looks good! Just some questions and fixes.

File: server/src/test/ZetaSqlWrapper.spec.ts (hunk 1 above)
Comment (pgrivachev) [reply server/src/test/ZetaSqlWrapper.spec.ts:63]: Unfortunately it doesn't work on compile time, I already tried to find solution
image
It requires to add more `?` so I decided to use `assert.ok`

**Review by fivetran-dmitryboykov — APPROVED** [review]
Looks good.

**Pull request merged**